testsuite/Makefile

```diff
@@ -19,6 +19,8 @@ NO_PRINT=`$(MAKE) empty --no-print-directory >/dev/null 2>&1 \
 
 FIND=find
 include ../config/Makefile
+# Borrowed from ../Makefile.tools
+BYTECODE_ONLY=[ "$(ARCH)" = "none" -o "$(ASM)" = "none" ]
```

Comment:
Seemed safer to copy this small definition rather than risk including Makefile.tools here